vm_supervisor/INSTANCES.md

```diff
@@ -0,0 +1,44 @@
+
+# Instance Messages
+
+I have added support to Instance message in the aleph-message repository, in this PR:
```

Comment:
Please use the passive form in documentation. "Support of Instance message in the aleph-message repository was added in this PR: ..." instead of "I have..."